### PR TITLE
Adds New Shuttle Chairs

### PR DESCRIPTION
**Pull request by fludd12**
:cl:Memager and Fel
imageadd: New chairs are on most shuttles! Finally, you can relax in style while escaping a metal death trap.
/:cl:

All sprite credit to Memager! I just added the object and mapped it in. Tell me if there are any other places you think I should add it to!

### CODE REVIEW

**Comment by fludd12** [comment]
Aaaaaa what went wrong

File: _maps/shuttles/arrival_box.dmm [hunk 1]
```diff
@@ -78,18 +78,8 @@
...
-/obj/structure/chair,
-/obj/machinery/light/small{
-	dir = 4
-	},
-/turf/open/floor/mineral/titanium/blue,
-/area/shuttle/arrival)
-"r" = (
-/obj/structure/chair{
+/obj/structure/chair/shuttle{
+	icon_state = "shuttle_chair";
```
Comment (SpaceManiac) [inline _maps/shuttles/arrival_box.dmm]: You can find&replace all of these lines away, since this var is set by the `/shuttle` subtype already.

**Comment by SpaceManiac** [comment]
I would recommend double-checking the diff, you seem to have deleted various wall-mounts (air alarms, flashers, lights ...) and turf decals.

**Comment by fludd12** [comment]
im just gonna redo this by replacing chair with chair/shuttle

**Comment by fludd12** [comment]
Ooooone moment please

**Comment by SpaceManiac** [comment]
Looks like you missed south-facing chairs; you may need to do another run for `/chair,` to `/chair/shuttle,`.

**Comment by fludd12** [comment]
the issue though, is chair/stool, etc.

**Comment by SpaceManiac** [comment]
where you were search/replacing on `{`, search/replace on `,`

**Comment by fludd12** [comment]
ohh, okay.

**Review by Cyberboss — APPROVED** [review]
10 years overdue

**Comment by cacogen** [comment]
looks bad

**Pull request merged**